docs/background.adoc

```diff
@@ -24,13 +24,13 @@ image::diag/domains.png[DLP Domains,align=center,title-align=center]
 
 *HPC, scientific computing, and ML training workloads* are typically characterized by very long application vector lengths, exhibiting a high degree of data-level parallelism.
 Systems targeting these workloads are generally designed and deployed at datacenter scale, with available data-level-parallelism exploited across the many processors in a multi-node cluster.
-While single-core efficiency and performance remain important for this domain, scalability towards multi-core, multi-node systems is a more critical concern.
+While single-core efficiency and performance remain important for this domain, scalability towards multi-core, multi-node systems are of more critical concern.
```

Comment:
The subject of this sentence is singular - "scalability", so "is" should be correct here